raiden/src/next_token/mod.rs

```diff
@@ -8,7 +10,7 @@ impl NextToken {
         Self(token.into())
     }
     pub fn into_attr_values(self) -> Result<super::AttributeValues, super::RaidenError> {
-        let decoded = match base64::decode(self.0) {
+        let decoded = match STANDARD.decode(self.0) {
```

Comment:
base64::decode and base64::encode is deprecated.
https://docs.rs/base64/0.21.4/base64/fn.decode.html